workloads/kube-burner/workloads/node-vm-density/vm.yml

```diff
@@ -0,0 +1,50 @@
+apiVersion: kubevirt.io/v1
+kind: VirtualMachine
+metadata:
+  labels:
+    name: {{.JobName}}
+  name: {{.JobName}}-{{.Iteration}}
+spec:
+  running: true
+  template:
+    metadata:
+      labels:
+        kubevirt-vm: {{.JobName}}-{{.Iteration}}
+    spec:
+      domain:
+        cpu:
+          cores: 1
+          sockets: 1
+          threads: 1
+        devices:
+          disks:
+            - disk:
+                bus: virtio
+              name: node-vm-density
```

Comment:
+            - disk:
+                bus: virtio
+              name: cloudinitdisk

Reply:
You have the cloudInitNoCloud section to set pw / config ssh so this template would fail w/out defining the corresponding `disk` (error was: `spec.template.spec.domain.volumes[1].name 'cloudinitdisk' not found`)